api/src/main/java/no/bibsys/web/DatabaseResource.java

```diff
@@ -361,6 +362,7 @@ public Response entitiesSummary(@HeaderParam(ApiKeyConstants.API_KEY_PARAM_NAME)
                     value = AwsApiGatewayIntegration.AWS_PROXY), }) })
     @SecurityRequirement(name = ApiKeyConstants.API_KEY)
     @RolesAllowed({ Roles.API_ADMIN, Roles.REGISTRY_ADMIN })
+    @Produces("application/json")

```

Comment:
Can we use something like javax.ws.rs.core.MediaType.APPLICATION_JSON? Possibly own class/enum if applicaiton/ld+json is to be supported.

Reply:
yes